client/src/includes/breadcrumbs.js

```diff
@@ -0,0 +1,77 @@
+export default function initCollapsibleBreadcrumbs() {
+  const breadcrumbsToggle = document.querySelector('[data-toggle-breadcrumbs]');
+  const breadcrumbLinks = document.querySelectorAll('[data-breadcrumb-link]');
```

Comment:
Mentioned in the template but technically none of those are links, so should be `data-breadcrumb-li` or `data-breadcrumb-item`, and same change for the variable.